File: Algorithms/Sorting/Counting_Sort_1/main.py
### Counting Sort 1 - Solution

from collections import Counter

def countingSort(arr):
    cnt = Counter(arr)
    idxCounter = (cnt[i] for i in range(100))
    print(*idxCounter) 

n = int(input())
arr = list(map(int, input().split()[:n]))
countingSort(arr)

File: Algorithms/Sorting/Counting_Sort_1/main.cpp
// Counting Sort 1 - Solution

#include <iostream>
#include <cstring>

void countingSort(const int n, const int arr[]) {
    int zeroArr[100];
    std::memset(zeroArr, 0, sizeof zeroArr);
    for (int i{}; i < n; i++)
        zeroArr[arr[i]]++;
    for (int j{}; j < 100; j++)
        std::cout << zeroArr[j] << " ";
}

int main() {
    int n;
    std::cin >> n;
    int arr[n];
    for (int i{}; i < n; i++)
        std::cin >> arr[i];
    countingSort(n, arr);

    return 0;
}